actionmetadata_test.go

```diff
@@ -107,6 +107,9 @@ func TestKnowledgeBase(t *testing.T) {
 					lintIssues = append(lintIssues, fmt.Sprintf("Reason must start with 'to '. It is currently %s in action-security.yml at %s", scope.Reason, filePath))
 					return nil
 				}
+				if len(scope.Reason) > 50 {
```

Comment:
@Devils-Knight please add a comment here, so we remember why this was done, e.g. `since the reason is added as a comment in the workflow file, limit the length to 50 to not clutter the workflow file`

Reply:
@Devils-Knight one of the action-security.yml is failing this test. Can you please update that as well?